pan/legacy.pan

```diff
@@ -1,3 +1,12 @@
 declaration template pan/legacy;
 
 type legacy_binary_affirmation_string = string with match(SELF, "^(yes|no)$");
+
+function is_yes_no_true_false = {
+    v = ARGV[0];
+    if (is_boolean(v)) return(true);
+    if (is_string(v) && match(v, "^(yes|no)$")) return(true);
```

Comment:
if we are not going to do anything, why do we even switch to `transitional_yes_no_true_false`?